<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test9Window</class>
 <widget class="QMainWindow" name="Test9Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test9 Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>798</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="minimumSize">
       <size>
        <width>1207</width>
        <height>780</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>310</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="gridLayoutWidget">
           <layout class="QGridLayout" name="leftLayout"/>
          </widget>
          <widget class="QWidget" name="gridLayoutWidget_2">
           <layout class="QGridLayout" name="rightLayout"/>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QScrollBar" name="verticalScrollBar">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <layout class="QGridLayout" name="bottomLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1225</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Old"/>
    <addaction name="actionLoad_New"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Old">
   <property name="text">
    <string>Load Old</string>
   </property>
  </action>
  <action name="actionLoad_New">
   <property name="text">
    <string>Load New</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
